InnerEye/ML/photometric_normalization.py

```diff
@@ -83,6 +83,10 @@ def transform(self, image: Union[np.ndarray, torch.Tensor],
             else:
                 mask = np.ones_like(image)
 
+        is3d = image.ndim == 3
```

Comment:
Could you add test coverage for this fix?

Reply:
Where should I test this? I can't find any tests for intensity normalisation. Also, what is our standard way to look at coverage?

Reply:
I can try the Codecov-GitHub integration for this repo, I think it's quite nice.

Reply:
Added coverage in 478698e47305510dd4c7fb878ea6cf30e6057123.